apps/services/schemas.py

```diff
@@ -24,17 +24,17 @@ class UserRequestExistenceOut(BaseOutSchema):
 
 
 class UserRequestBookingHostelOut(BaseOutSchema):
-    full_name: str
+    full_name: Dict[str, str]
     user_id: int
     faculty_name: str
     university_id: int
     short_university_name: str
-    rector_full_name: str
+    rector_full_name: Dict[str, str]
```

Comment:
Should be an embedded schema, that describes this structure

Reply:
example?

Reply:
fixed